src/main/webapp/app/modules/auth/complete-profile/complete-profile.reducer.ts

```diff
@@ -65,36 +65,36 @@ export default (state: CompleteProfileState = initialState, action): CompletePro
         updateSuccess: true,
         loading: false,
         errorMessage: null,
-        userProfile: action.payload.data
+        userProfile: action?.payload?.data ?? defaultValue,
```

Comment:
please do an `npm install` before running the code.
There is an update on `typescript` and `prettier` package.

I'm quite sure that the code is fine because the GitAction Ci doesn't complaint

Reply:
fixed in 3fc02c7c262735ceef449adfbca62fa9ecadd061